<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>messenger</class>
 <widget class="QMainWindow" name="messenger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="messagesInput">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>490</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="tabStopWidth">
     <number>80</number>
    </property>
    <property name="placeholderText">
     <string>Введите сообщение..</string>
    </property>
   </widget>
   <widget class="QPushButton" name="messagesPush">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>490</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Отправить</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="messagesBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>441</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="nameChat">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Messenger</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>298</x>
      <y>71</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name">
    <property name="geometry">
     <rect>
      <x>367</x>
      <y>70</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
